### DISCUSSION TITLE
Icon request:  surfing

### DISCUSSION THREAD

**Discussion by jameswilson** [FortAwesome/Font-Awesome#21142] [Icon Requests]
Not sure if Font-Awesome has plans to delve into the water sports arena (or already has), but some surf-related icons would be cool:

Ideas include:
- surfboard: longboard, shortboard, fish (this is a specific recognizable shape of a board),
- humanistic: surfer on surfboard, surfer on wind surfboard, surfer on stand up paddle,
- general ocean landscape: palm tree on sandy island,
- wave conditions:  small wave, medium size wave, giant wave, two waves,

**Comment by jameswilson** [comment]
Tangentially related to icon-beach proposal here: #1145

**Comment by arcidodo** [comment]
Hi, we are working on a project where we need surf board icons. are there plans to make these icons?

the icons that we need are:
- Wave surfing icon
- Wind surf icon
- Kite surf icon

Thanks.

**Comment by jaxsunsports** [comment]
+1

**Comment by mdepecker** [comment]
+1

**Comment by silverakyle** [comment]
+1

**Comment by tonijs** [comment]
+1

**Comment by aroniotis** [comment]
+1

**Comment by shmaster** [comment]
+1

**Comment by cnrblm** [comment]
+1

**Comment by BenasPaulikas** [comment]
+9999

**Comment by ahoendgen** [comment]
+1

**Comment by smyl3s** [comment]
+1 surfing / wave

**Comment by VSpock** [comment]
+1

**Comment by SeanFmcdonnell** [comment]
+1

**Comment by adimoraret** [comment]
+1

**Comment by tcjennings** [comment]
Also:
- a hang-loose hand

**Comment by jameswilson** [comment]
^ like it, aka, "icon-shaka" 🤙

**Comment by morgzz** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🏄 [U+1F3C4](https://emojipedia.org/emoji/%F0%9F%8F%84/) Person Surfing
Also known as `Surf`, `Surfing` and `Surfer`.

**Comment by jarek7410** [comment]
+!

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Reply by nmooradian-lacdbh** [reply]
Is this icon still on the list to be created?

**Comment by JudyAnnGorre** [comment]
Record_2024-07-12-00-22-11.mp4
<https://drive.google.com/file/d/11WJPz3nCuiR8iwtdhwX8hin6iJWXim9O/view?usp=drivesdk>
I'm sorry but what are these emails for? There are more than 500+ of these.

**Comment by sensibleworld** [comment]
@JudyAnnGorre I might check your settings in regards to the Font Awesome repo. It's possible you're getting notifications for everything.
